aperture.go

```diff
@@ -452,10 +468,20 @@ func initTorListener(cfg *config, etcd *clientv3.Client) (*tor.Controller, error
 func createProxy(cfg *config, challenger *LndChallenger,
 	etcdClient *clientv3.Client) (*proxy.Proxy, error) {
 
+	var seed *string
```

Comment:
`nil` strings are used rarely in golang. It's totally fine to just use `var seed string` and later do a `if seed == ""` comparison.